ui-components/src/locales/general.json

```diff
@@ -697,6 +697,16 @@
   "errors.rateLimitExceeded": "Rate limit exceeded, try again later.",
   "errors.requiredFieldError": "This field is required.",
   "errors.requiredFieldsError": "These fields are required",
+  "errors.minGreaterThanMaxOccupancyError": "Min Occupancy must be less than or equal to Max Occupancy",
```

Comment:
We'll need to add these strings in the partners override file since they only exist in partners